manifests/cf-manifest/env-specific/cf-staging.yml

```diff
@@ -3,5 +3,5 @@ meta:
   cell:
     instances: 6
   elasticsearch_master:
-    disk_size: 307400
+    disk_size: 460800
```

Comment:
I glanced at the metrics in Datadog:

* A 50% increase in disk should take the current 76% usage down to ~51%.
* We alert at 75% usage. That gives 24% headroom before the next alert.
* Disk usage has grown by 19% in 3 months, which is roughly 6.3% per month.
* A 50% increase in disk should take the growth rate down to 6.3/1.5 ≈ 4.2% per month.
* That 24% headroom will take 24/4.2 ≈ 5.7 months to use up.

This is fine provided we expect staging to be using Logit in August?

Reply:
I think that's fine. I would expect that we will be using Logit by then.